Anabi/Features/Decision/Models/DecisionDetails.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Anabi.Features.Decision.Models
+{
+    public class DecisionDetails
```

Comment:
personally I naming classes by XSummary and X (which contains the details), instead of XDetails. but this is just a preference. I think we should set a rule for naming classes.